### PR TITLE
Update MaxText config for Llama2 7B on GPUs.

### PR DESCRIPTION
**Pull request by yangyuwei**

### CODE REVIEW

**Review by michelle-yooh — APPROVED** [review]

File: MaxText/configs/a3/llama_2_7b/16vm.sh [hunk 1]
```diff
@@ -1,35 +1,47 @@
...
+ --xla_disable_hlo_passes=rematerialization"
 
-# 16 nodes
-python MaxText/train.py MaxText/configs/base.yml run_name=$RUN_NAME hardware=gpu \
+# 16 node, DCN_DP=16, ICI_FSDP=8
+python MaxText/train.py MaxText/configs/base.yml hardware=gpu \
     steps=30 dcn_data_parallelism=16 ici_fsdp_parallelism=8 per_device_batch_size=4 max_target_length=4096 model_name=llama2-7b \
     enable_checkpointing=false attention=cudnn_flash_te remat_policy=minimal_flash use_iota_embed=true scan_layers=false \
-    dataset_type=synthetic async_checkpointing=false base_output_directory=gs://runner-maxtext-logs profiler=xplane
+    dataset_type=synthetic async_checkpointing=false base_output_directory=$OUTPUT_PATH logits_dot_in_fp32=false profiler=xplane
```
Comment (gobbleturk) [inline MaxText/configs/a3/llama_2_7b/16vm.sh:47]: Have you tested the logits_dot_in_fp32 change, e.g. via a golden logits test?

**Review by gobbleturk — APPROVED** [review]

**Review by NinaCai — APPROVED** [review]

File: MaxText/configs/a3/llama_2_7b/16vm.sh (hunk 1 above)
Comment (yangyuwei) [reply MaxText/configs/a3/llama_2_7b/16vm.sh:47]: Actually, we have been using `logits_dot_in_fp32=false` for both A3 and A3+ benchmarking, but this flag was missing when we added these scripts. Would you point me which golden logits test shall we run?

Comment (gobbleturk) [reply MaxText/configs/a3/llama_2_7b/16vm.sh:47]: hmm I do think a golden logits test makes sense to include, it will be giving coverage to both GPUs in general and to these specific choices
